### DISCUSSION TITLE
Add amazon wishlist to the social icons

### DISCUSSION THREAD

**Discussion by gerrited** [hugo-sid/hugo-blog-awesome#155] [Feature requests]
Hi everyone!

The theme already contains a good selection of social icons. The only thing I'm missing is an icon for my amazon wishlist. I am in the process of migrating my website gerrit.codes to this theme and the old theme is simply using the amazon logo as an icon. Maybe that is also an option for this theme.

TIA Gerrit

**Comment by hugo-sid** [comment]
Hi @gerrited. Thanks for the feature request.

Adding amazon logo as an icon isn't a problem.
